.gitignore

```diff
@@ -1,5 +1,7 @@
-node_modules
-coverage
+node_modules/
+coverage/
 .DS_Store
 yarn.lock
 .vscode
+.idea/
+*.iml
```

Comment:
Minor : check if you set up editorconfig support; it's a plugin in intellij if I remember. According to `.editorconfig`, files end with a newline